theano/compile/tests/test_function_module.py

```diff
@@ -809,7 +826,7 @@ def pers_load(id):
         fp.close()
         p = pickle.Unpickler(fp2)
         p.persistent_load = pers_load
-        f2 = p.load()
+        # f2 = p.load()
```

Comment:
This should not be commented out, but rather called without assigning it to `f2`.